.github/workflows/test.yml

```diff
@@ -153,16 +155,16 @@ jobs:
       - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:
-            toolchain: nightly
-            profile: minimal
-            target: ${{ matrix.rust_target }}
-            default: true
+          toolchain: nightly
+          profile: minimal
+          target: ${{ matrix.rust_target }}
+          default: true
+      - name: Install cross
+        run: |
+          cargo install cross --git https://github.com/cross-rs/cross
```

Comment:
Why `--git`? I guess they haven't produced any releases recently, is that why?

Reply:
I've read it from [README](https://github.com/cross-rs/cross?tab=readme-ov-file#installation).
Anyway, I can remove it, if it needs.